packages/SwingSet/test/virtualObjects/collection-slots/test-collection-slots.js

```
@@ -10,17 +10,14 @@ import {
   makeSwingsetController,
 } from '../../../src/index.js';
 
+import { kunser } from '../../../src/lib/kmarshal.js';
+
 function bfile(name) {
   return new URL(name, import.meta.url).pathname;
 }
 
 function getImportSensorKref(impcapdata, i) {
-  const body = JSON.parse(impcapdata.body);
-  const value = body[i];
-  if (typeof value === 'object' && value['@qclass'] === 'slot') {
-    return ['slot', impcapdata.slots[value.index]];
-  }
-  return value;
+  return ['slot', `${kunser(impcapdata)[i]}`];
```

another place where `krefOf` would be safer

And if `krefOf` throws instead of returning `undefined`, then `getImportSensorKref` could be change to return just the kref, not `['slot', kref]`, and the callers elsewhere in this file could be simpler.